docs/operations/deploy_common_instancetypes.md

```diff
@@ -0,0 +1,29 @@
+# Deploy common-instancetypes
+
+The [`kubevirt/common-instancetypes`](https://github.com/kubevirt/common-instancetypes) provide a set of [instancetypes and preferences](../virtual_machines/instancetypes.md) to help create KubeVirt [`VirtualMachines`](http://kubevirt.io/api-reference/main/definitions.html#_v1alpha1_virtualmachine).
```

Comment:
Why not v1 on the VM? https://kubevirt.io/api-reference/main/definitions.html#_v1_virtualmachine

Reply:
Good question, that was copied and pasted.